src/openssl/ssl/context.cr

```diff
@@ -40,17 +40,13 @@ abstract class OpenSSL::SSL::Context
     # context = OpenSSL::SSL::Context::Client.new
     # context.add_options(OpenSSL::SSL::Options::NO_SSL_V2 | OpenSSL::SSL::Options::NO_SSL_V3)
     # ```
-    #
```

Comment:
These changes are duplicate of #14656

Reply:
Yes, this PR is supposed to go on top of that.

@ysbaddaden I usually mark such PRs as *Draft* until the PRs they depend on are merged. That makes it obvious that they're not ready yet.

Reply:
Yes, but they were ready for review, and I specified "follow up to", but I should have specified that the N first commits are from the other PR.